e2e/_setup.ts

```diff
@@ -1,4 +1,6 @@
 
+import { test, expect } from '@playwright/test';
```

Comment:
SonarClouding mukaan "test":n import on turha tässä.
https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=1029&id=City-of-Helsinki_haitaton-ui&open=AZQsaLXcp3pLVeoorkB3